src/main/java/io/hops/util/Hops.java

```diff
@@ -1898,6 +1905,9 @@ public static void createFeaturegroup(
     if (numClusters == null) {
       numClusters = 5;
     }
+    if(jobName == null || jobName.isEmpty()){
```

Comment:
@Limmen we usually call the google `Strings` library for such checks, [example](https://github.com/logicalclocks/hops-util/blob/release-0.8/src/main/java/io/hops/util/KafkaProperties.java#L85).

Reply:
Done